examples/modem_reset.pl

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/perl
```

Comment:
the scripts are kind of unfinished and untestet because I had to strip out code, which is only suitable for our OS enviroment. 
I used the reset script if the modem wasn't respond to the rpc calls at all. 
I think the perl GTK3 program should be redone in python. i used it to trigger a NM connect and disconnect from the system tray. 
I just added them to the examples directory, because it could be a source for others.